charts/app-config/values-integration.yaml

```diff
@@ -1432,6 +1432,8 @@ govukApplications:
         alb.ingress.kubernetes.io/load-balancer-name: www-origin
         alb.ingress.kubernetes.io/healthcheck-path: /readyz
         alb.ingress.kubernetes.io/load-balancer-attributes: access_logs.s3.enabled=true,access_logs.s3.bucket=govuk-integration-aws-logging,access_logs.s3.prefix=elb/www-origin-eks
+        alb.ingress.kubernetes.io/conditions.{{ .Release.Name }}: >
+          [{"field":"host-header","hostHeaderConfig":{"values":["www.{{ .Values.externalDomainSuffix }}", "www.{{ .Values.publishingServiceDomainSuffix }}"]}}]
```

Comment:
Do we not also need www-origin in these?

Reply:
The ALB adds the host specified in the rules to the list allowed host headers - so should be okay.

Reply:
(oh if you mean "www-origin.publishing.service.gov.uk"? Think that'd send traffic to the router ELB in the old infra anyway - we wouldn't be splitting that traffic)

Reply:
Aha cool 👍